documentation/DATA_FORMAT.md

```diff
@@ -196,7 +199,108 @@ The manifest file would look something like this:
 
 </details>
 
-### 2. Tracks
+### 2. Backdrops (optional)
+
+Multiple sets of **backdrop images** can be included in the manifest, which will be shown behind the colored objects in the UI. Each backdrop image set is defined by a JSON object with a `name`, `key`, and `frames`.
+
+The `key` must be unique across all backdrop image sets, and must only contain lowercase alphanumeric characters and underscores.
+
+`frames` is a list of **relative image paths** corresponding to each frame in the time series. Each set must have **one backdrop image for every frame in the time series**, and they must all be listed in order in the manifest file.
+
+`manifest.json:`
+
+```txt
+{
+    ...
+    "backdrops": [
+        {
+            "name": <backdrop name>,
+            "key": <backdrop key>, // Must be unique across backdrops. Lowercase alphanumeric characters and underscores only.
```

Comment:
maybe worth a general note somewhere near the top that several fields have a "key" attribute which MUST be unique and alphanumeric+underscore.